db/migrations/20200407102623_create_log_item_update.sql

```diff
@@ -0,0 +1,17 @@
+-- +goose Up
+CREATE TABLE maker.log_item_update
+(
+    id         SERIAL PRIMARY KEY,
+    log_id     BIGINT  NOT NULL REFERENCES public.event_logs (id) ON DELETE CASCADE,
+    header_id  INTEGER NOT NULL REFERENCES public.headers (id) ON DELETE CASCADE,
+    offer_id   INT,
+    UNIQUE (header_id, log_id)
```

Comment:
Wondering if we might want to add the contract's address_id to this table if it's populated from multiple contracts

Reply:
oh yeah, good call!